src/index.html

```diff
@@ -2,16 +2,174 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
+    <meta name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="displayInput">
+        <legend>
+          Personal information
+        </legend>
+        <label for="surname" >
+          Surname:
+          <input
+          type="text"
+          id="surname"
+          >
+        </label>
+        <label for="name" class="margin10Px">
+          Name:
+          <input
+          type="text"
+          id="name"
+          >
+        </label>
+        <label for="oldAreYou" class="margin10Px">
+          How old are You?
+          <input
+          type="number"
+          id="oldAreYou"
+          value="12"
+          class="oldAreYou"
+          >
```

Comment:
```suggestion
          <input
            type="number"
            id="oldAreYou"
            value="12"
            class="oldAreYou"
          >
```
Fix your code style everywhere according to [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)